### PR TITLE
Feature/dropdown

### PR DESCRIPTION
**Pull request by riqthedev**
## Description

Added dropdown menu for tech stack field

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,15 +6,18 @@
   "repository": "github:Lambda-School-Labs/labs-spa-starter",
   "dependencies": {
     "@ant-design/icons": "^4.7.0",
+    "@ant-design/pro-form": "^1.52.13",
+    "@ant-design/pro-table": "^2.62.7",
     "@craco/craco": "^5.6.4",
     "@okta/okta-react": "^3.0.2",
     "@okta/okta-signin-widget": "^4.1.2",
-    "antd": "^4.16.13",
+    "antd": "^4.18.4",
```
Comment (MrJacobSullivan) [inline package.json:14]: Why change the version of this package?

File: package.json [hunk 2]
```diff
@@ -6,15 +6,18 @@
...
     "axios": "^0.21.1",
     "craco-less": "^1.17.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.26.1",
     "plotly.js": "^1.54.6",
+    "rc-field-form": "^1.22.1",
```
Comment (MrJacobSullivan) [inline package.json:20]: Why the new package? Is it worth the technical debt?

File: src/components/pages/FooterLanding/FooterLanding.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+//             to={`/landing`}
+//             alt="underdog devs logo"
+//             height="68"
+//             style={{ marginLeft: '1vw' }}
+//           />
+//         </Col>
+//       </Row>
+//     </Footer> */}
+//   </>
+// );
```
Comment (MrJacobSullivan) [inline src/components/pages/FooterLanding/FooterLanding.js:134]: This commented code should be deleted.

File: src/components/pages/LandingPage/LandingPage.css [hunk 4]
```diff
@@ -0,0 +1,362 @@
+/*Section 1*/
+.landInfo1 {
```
Comment (MrJacobSullivan) [inline src/components/pages/LandingPage/LandingPage.css:2]: Why the combination of CSS and styled-components?

**Review by tinaxgao — CHANGES_REQUESTED** [review]
It seems like there are too many updates in this PR, so maybe remove changes to the file that has conflicts.

**Review by MrJacobSullivan — CHANGES_REQUESTED** [review]
It looks like your local Prettier config is overriding the project Prettier config. Thus, when you run `npm run prettier` it conforms the entire project to your settings. This PR makes changes to 43 files. I think you should rethink the scope of this PR. You are making substantial changes to the landing page even though the PR is called `dropdown`.

**Review by cainpalmer — CHANGES_REQUESTED** [review]
Could you split this PR into multiple ones? It would make it easier to review the code and check for feedback!

**Comment by paulstgermain** [comment]
Merging this PR - the excess in files changed are actually coming from having this branch be brought up to speed with changes it didn't have from `main` as a side effect of the rebase flow.

**Pull request merged**